create-publication-from-scopus/src/main/java/no/sikt/nva/scopus/conversion/model/cristin/Affiliation.java

```diff
@@ -0,0 +1,68 @@
+package no.sikt.nva.scopus.conversion.model.cristin;
```

Comment:
Siden dette har blitt kopier fra nva-cristin-service, burde denne vært tatt inni  nva-datamodel slik at vi kan vi kan importere dette istedet for copy-pase.